pytest/tests/mocknet/helpers/neard_runner.py

```diff
@@ -172,15 +174,20 @@ def parse_binaries_config(self):
                     # TODO: maybe it could make sense to allow this, meaning don't run any binary
                     # on this node until the network reaches that epoch, then we bring this node online
                     sys.exit(
-                        f'config should contain one binary with epoch_height 0')
+                        f'config should contain one binary with epoch_height 0'
+                    )
             else:
                 if epoch_height == last_epoch_height:
-                    sys.exit(f'repeated epoch height in config: {epoch_height}')
+                    sys.exit(
+                        f'repeated epoch height in config: {epoch_height}')
             last_epoch_height = epoch_height
             binaries.append({
-                'url': b['url'],
-                'epoch_height': b['epoch_height'],
-                'system_path': self.home_path('binaries', f'neard{i}')
+                'url':
+                b['url'],
+                'epoch_height':
+                b['epoch_height'],
+                'system_path':
+                self.home_path('binaries', f'neard{i}')
```

Comment:
Some auto formatting kicked in, do we have a recommended formatter?

Reply:
not sure, but following what is done in `scripts/formatting`, I usually run: `python3 -m yapf -pir {filename}`